osu.Game/Skinning/LegacySkin.cs

```diff
@@ -368,7 +368,7 @@ protected override void ParseConfigurationStream(Stream stream)
                                 {
                                     songProgress.Anchor = Anchor.TopRight;
                                     songProgress.Origin = Anchor.CentreRight;
-                                    songProgress.X = -accuracy.ScreenSpaceDeltaToParentSpace(accuracy.ScreenSpaceDrawQuad.Size).X - 10;
+                                    songProgress.X = -accuracy.ScreenSpaceDeltaToParentSpace(accuracy.ScreenSpaceDrawQuad.Size).X - 20;
```

Comment:
I have zero idea why, but on my device, none of the skins linked in the issue look as good as they do on the gifs in OP. For me this works better tweaked to

```suggestion
                                    songProgress.X = -accuracy.ScreenSpaceDeltaToParentSpace(accuracy.ScreenSpaceDrawQuad.Size).X - 18;
```

I can't judge why this might be because I don't have any full screenshot from your testing. I'll put mine, though.

| | this PR | my suggestion |
| :-: | :-: | :-: |
| skin 1 | ![osu_2023-07-19_23-02-17](https://github.com/ppy/osu/assets/20418176/27194942-77b1-4fd4-a42c-4e7b7bbcc078) | ![osu_2023-07-19_23-03-35](https://github.com/ppy/osu/assets/20418176/a46c411d-5f79-4c3f-b670-3c34b8ff4528) |
| skin 2 | ![osu_2023-07-19_23-02-28](https://github.com/ppy/osu/assets/20418176/12241cd0-4cfe-42d6-bc52-ff44b0af0c57) | ![osu_2023-07-19_23-03-44](https://github.com/ppy/osu/assets/20418176/a764669b-1f09-4cb6-affd-710192b69b65) |
| skin 3 | ![osu_2023-07-19_23-02-37](https://github.com/ppy/osu/assets/20418176/db176872-6d07-4489-b790-8e164759ef95) | ![osu_2023-07-19_23-03-56](https://github.com/ppy/osu/assets/20418176/465150b5-9f6a-432d-8d80-0fa72aa1a596) |
| skin 4 | ![osu_2023-07-19_23-02-50](https://github.com/ppy/osu/assets/20418176/2bba2e90-fdbc-4159-a38d-0daef9c641dd) | ![osu_2023-07-19_23-04-05](https://github.com/ppy/osu/assets/20418176/ed47524d-fc61-40f2-9474-493d9b252d49) |

All screens taken at 1920x1080, so I'd like to think there's no stupid business there and it's pure 16:9... No UI scaling involved, I checked.

Reply:
Please go with your changes. I was ballparking the resolution due to https://github.com/ppy/osu-framework/pull/5937 not working, so your definitive screen size is likely 100% more correct.